gulpfile.js

```diff
@@ -10,42 +10,60 @@ var del = require('del');
 
 require('babel-core/register');
 
-var src = [
-  'src/shady-css/common.js',
-  'src/shady-css/node-factory.js',
-  'src/shady-css/node-visitor.js',
-  'src/shady-css/stringifier.js',
-  'src/shady-css/token.js',
-  'src/shady-css/tokenizer.js',
-  'src/shady-css/parser.js',
-  'src/*.js'
+var src = ['src/**/*.js'];
+var dist = 'dist';
+var concatOrder = [
```

Comment:
Because UMD module definition is synchronous (unless an async loader is available), so the concatenated file needs definitions to be in the right order.